libraries/eosiolib/sdk/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+cmake_minimum_required(VERSION 3.16)
+project(clsdk)
```

Comment:
Are users expected to pull this in using `add_subdirectory`?  For a pre-built package, support for `find_package`, i.e. clsdk-config.cmake, would be more idiomatic.

Reply:
I did some experimenting and found a way to force the `find_package` approach to work by setting `clsdk_DIR`. `find_package`'s normal rules (e.g. searching through `CMAKE_PREFIX_PATH`) are borked by a poorly-documented and poorly-thought-out (IMO) feature: `find-package` prefixes paths in `CMAKE_PREFIX_PATH` with `CMAKE_SYSROOT`.